src/renderer.tsx

```diff
@@ -141,7 +141,6 @@ export default function (context): void {
 
 		return {
 			...newSiteEnvironmentProps,
-			onGoBack: () => LocalRenderer.sendIPCEvent('goToRoute', LOCAL_ROUTES.ADD_SITE_CREATE_NEW),
```

Comment:
Don't need this really since we're already passing onGoBack in newSiteEnvironmentProps with the correct histoy.goBack function